cli/src/component/main_view.rs

```diff
@@ -71,8 +72,17 @@ impl Input for MainView {
             state.focus_on(focus::TERMINATION);
             state.update_state();
 
+            // Spawn a new thread to exit the process after 30s if it has not already exited
             if !is_docker_running() {
+                let _ = disable_raw_mode();
                 std::process::exit(0);
+            } else {
+                std::thread::spawn(|| {
+                    std::thread::sleep(std::time::Duration::from_secs(60));
```

Comment:
Why so long?

Reply:
As per above, we need ample time for the application to shutdown normally before the failsafe kicks in forcefully.